packages/components/checkbox/src/use-checkbox.ts

```diff
@@ -309,7 +309,15 @@ export function useCheckbox(props: UseCheckboxProps = {}) {
   const getInputProps: PropGetter = useCallback(() => {
     return {
       ref: mergeRefs(inputRef, ref),
-      ...mergeProps(inputProps, focusProps),
+      ...mergeProps(
+        inputProps,
+        focusProps,
+        filterDOMProps(otherProps, {
```

Comment:
This does also mean that the `id` prop now gets passed to the input as well, which seems like the correct behaviour